readthedocs/rtd_tests/tests/test_notifications.py

```diff
@@ -144,6 +145,9 @@ class TestNotification(SiteNotification):
             success_level = INFO_NON_PERSISTENT
 
         user = fixture.get(User)
+        # Setting the primary and verified email address of the user
+        email = fixture.get(EmailAddress, user=user, primary=True, verified=True)
```

Comment:
I don't understand how this works. Where are we testing that the user does not have a verified email and where that it does have it?

Reply:
I added the primary and verified email for the test user because this line
https://github.com/rtfd/readthedocs.org/blob/cd6e82118541b1612b24540efebdefb7af92827e/readthedocs/rtd_tests/tests/test_notifications.py#L155

tests that there is only one persistent message but because the email of the test user is not verified, it adds another notification which is added in this PR.